app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= stylesheet_link_tag 'sign', media: 'all', 'data-turbolinks-track': 'reload' %>
+<div class="form-container">
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="form-group mb-3">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+  <div class="form-group mb-3">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="form-group mb-3">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="form-group mb-3">
+    <%= f.label :password_confirmation %><br />
```

Comment:
- [ ] As stated by the previous reviwer, kindly edit the form fields for `sign up` as you beautifully did for `login`.
- Currently you have labels instead of placeholders which was pointed out by the previous reviwer.
- Good job 👏 

### Logi done welll 👏:
![image](https://github.com/melkamu12/microverse_blogapp/assets/50990925/eee2d287-511c-4685-83a9-a9e1c9343ac4)

### Sign-up needing chnages:
![image](https://github.com/melkamu12/microverse_blogapp/assets/50990925/38041804-0753-4b54-a727-efe2b0fc5d05)

Reply:
Hi @melkamu12 nice job :clap: :clap: